test/internet/test-dns.js

```diff
@@ -390,6 +390,15 @@ TEST(function test_lookup_ipv6_explicit_object(done) {
 
 
 TEST(function test_lookup_ipv6_hint(done) {
+  // FreeBSD does not support V4MAPPED flag.
+  if (process.platform === 'freebsd') {
+    console.log(
+      '1..0 # Skipped: test_lookup_ipv6_hint is disabled on FreeBSD.'
```

Comment:
@jbergstroem Unfortunately, nope. It will not know the specific test :( I thought that if we specified the actual reason, it would be better to understand why it is skipped.

Now there is another interesting thing here. If we skip more than one tests in a single file, then there will be two skip TAP lines. I wonder how the output will be displayed.